libs/components/tabs/src/lib/modules/tabs/tabset.service.ts

```diff
@@ -45,13 +54,79 @@ export class SkyTabsetService {
       !this.tabIndexesEqual(value, this.currentActiveTabIndex)
     ) {
       this.currentActiveTabIndex = value;
+
       /* istanbul ignore else */
       if (config.emitChange) {
         this._activeTabIndex.next(value);
       }
+
+      this.setFocusedTabBtnIndex(value);
+    }
+  }
+
+  public focusNextTabBtn(tabButtons: TabButtonViewModel[]): void {
+    const currentTabArrayIndex = this.tabs.findIndex((tab) =>
+      this.tabIndexesEqual(tab.tabIndex, this.currentFocusedTabBtnIndex)
+    );
+
```

Comment:
`getNextTabArrayIndex` loops back to the beginning--when it reaches the last item, it returns 0 as the next index.

This loop starts with the very next tab and keeps going until it has either found a tab that isn't disabled to focus on next, or it reaches the tab it was already focused on, which means there are no other buttons to move focus to.